### PR TITLE
APP UI

### PR DESCRIPTION
**Pull request by shaqayq**
In this part of the project, I have created a form of UI for the school library app.
I have created a `main.rb` as an executable file and `app.rb ` for the app class that contains all app`s functions like:

- book_list
- people_list
- create_book
- create_person(create_teacher , create_student)
- create_rental
- rental_list_by_id
- exit.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+require_relative 'app'
+class Main
+  def initialize
+    @app = App.new
+  end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
```
Comment (FrederickMih) [inline main.rb]: - [ ] Kindly fix this Linter error. Do not disable them. Linters help to detect and flag programming errors.

File: teacher.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 require_relative 'Person'
```
Comment (FrederickMih) [inline teacher.rb]: - [ ] Please always make sure to double check your app before submission. You are not loading the required file here. It should be `require_relative 'person'` and not `require_relative 'Person'`. Also, correct the same error in the Student class.


Screenshot from 2022-09-30 11-45-29

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @shaqayq , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- PR has a title and summary :+1: 
- Correct GitHub flow :+1: 

## Required Changes ♻️

- [ ] The README.md file is missing the License section. Kindly include this section in order to render your README.md file professional :+1: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @shaqayq  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: teacher.rb (hunk 2 above)
Comment (shaqayq) [reply teacher.rb]: @FrederickMih!
Thank you for reviewing my code I will change it.

**Pull request merged**